addons/main/config.cpp

```diff
@@ -29,6 +29,7 @@ class CfgPatches {
             "cba_diagnostic",
             "cba_help",
             "cba_jr",
+            "cba_jam",
```

Comment:
Are these sorted? ja < jr

Reply:
It's not sorted alphabetically, but... thematically? Anyway, JR is the older one, so I put it first of these two.